rest/config.go

```diff
@@ -59,15 +59,16 @@ const (
 
 // Bucket configuration elements - used by db, index
 type BucketConfig struct {
-	Server         *string `json:"server,omitempty"`      // Couchbase server URL
-	DeprecatedPool *string `json:"pool,omitempty"`        // Couchbase pool name - This is now deprecated and forced to be "default"
-	Bucket         *string `json:"bucket,omitempty"`      // Bucket name
-	Username       string  `json:"username,omitempty"`    // Username for authenticating to server
-	Password       string  `json:"password,omitempty"`    // Password for authenticating to server
-	CertPath       string  `json:"certpath,omitempty"`    // Cert path (public key) for X.509 bucket auth
-	KeyPath        string  `json:"keypath,omitempty"`     // Key path (private key) for X.509 bucket auth
-	CACertPath     string  `json:"cacertpath,omitempty"`  // Root CA cert path for X.509 bucket auth
-	KvTLSPort      int     `json:"kv_tls_port,omitempty"` // Memcached TLS port, if not default (11207)
+	Server                                  *string `json:"server,omitempty"`                                        // Couchbase server URL
+	DeprecatedPool                          *string `json:"pool,omitempty"`                                          // Couchbase pool name - This is now deprecated and forced to be "default"
+	Bucket                                  *string `json:"bucket,omitempty"`                                        // Bucket name
+	Username                                string  `json:"username,omitempty"`                                      // Username for authenticating to server
+	Password                                string  `json:"password,omitempty"`                                      // Password for authenticating to server
+	CertPath                                string  `json:"certpath,omitempty"`                                      // Cert path (public key) for X.509 bucket auth
+	KeyPath                                 string  `json:"keypath,omitempty"`                                       // Key path (private key) for X.509 bucket auth
+	CACertPath                              string  `json:"cacertpath,omitempty"`                                    // Root CA cert path for X.509 bucket auth
+	KvTLSPort                               int     `json:"kv_tls_port,omitempty"`                                   // Memcached TLS port, if not default (11207)
+	MaxConcurrentViewQueryOperationsPerNode *int    `json:"max_concurrent_view_query_operations_per_node,omitempty"` // Max concurrent view / query ops
```

Comment:
Should be renamed to max_concurrent_query_ops, since it's not limited to views, and it's not per node.